samples/SocketsSample/LineInvocationAdapter.cs

```diff
@@ -1,91 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
It was in SocketSample - looks like it has been removed (by me LOL). In that case I think this: https://github.com/aspnet/SignalR/blob/dev/samples/SocketsSample/wwwroot/hubs.html#L10-L13 can be removed.